scipy/stats/_discrete_distns.py

```diff
@@ -258,8 +258,29 @@ class nbinom_gen(rv_discrete):
     for :math:`k \ge 0`, :math:`0 < p \leq 1`
 
     `nbinom` takes :math:`n` and :math:`p` as shape parameters where n is the
-    number of successes, :math:`p` is the probability of a single success, 
-    and :math:`1-p` is the probability of a single failure. 
+    number of successes, :math:`p` is the probability of a single success,
+    and :math:`1-p` is the probability of a single failure.
+
+    Another common parameterization of the negative binomial distribution is
+    in terms of the mean number of failures :math:`\mu` to achieve :math:`n`
+    successes. The mean :math:`\mu` is related to the probability of success
+    as
+
+    .. math::
+
+       p = \frac{n}{n + \mu}
+
+    The number of successes :math:`n` may also be specified in terms of a
+    "dispersion", "heterogeneity", or "aggregation" parameter :math:`\alpha`,
+    which relates the mean :math:`\mu` to the variance :math:`\sigma^2`,
+    e.g. :math:`\sigma^2 = \mu + \alpha \mu^2`. Regardless of the convention
+    used for :math:`\alpha`,
```

Comment:
I don't think these comments about alpha are helpful, and could be dropped.  The equations that follow give p and n in terms of mu and sigma, so I don't see why these are introduced with "Regardless of the convention used for alpha,".  I think this could be simplified to something like
```suggestion
    The parameters :math:`n` and :math:`p` may be expressed in terms of the
    mean :math:`\mu` and variance :math:`\sigma^2` as
```

Reply:
@afogarty85 do you agree with @WarrenWeckesser?

It seems to me that this would no longer fully address your issue, as you know wanted to know the relationship between the mean/dispersion parameter parameterization and the `p`/`n` used by `nbinom`. 

The intent of this part was to help readers make the connection between the dispersion parameter and the variance if they are not used to thinking about the variance of the distribution.

Reply:
OK, I commented as someone not familiar with the dispersion parameter, and the text seemed too vague to be useful.  I read a bit more about the distribution, and I see that the mean mu and dispersion alpha are another widely used parametrization of the negative binomial distribution.  So it seems like we could say even more, to help answer the question "How do I translate between (n, p) and (mu, alpha)?", but that might end up being more than we want in this docstring.  So I'm fine with this as it is.

Reply:
I appreciate all the time and help here.

Reply:
@WarrenWeckesser if you're fine with this as is again, and you were planning to merge before, I thought I'd ping you for a merge now.